release-notes/opendistro-for-elasticsearch-anomaly-detection-kibana-plugin.release-notes-1.12.0.0.md

```diff
@@ -0,0 +1,23 @@
+## Version 1.12.0.0 Release Notes
+
+Compatible with Kibana 7.10.0
+
+### Enhancements
+
+* Fix vulnerability caused by old version formik. Issue: #333 ([#333 (#334](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/333 (#334))
+* change side bar position for anomly detection plugin ([#335](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/335))
+
+### Bug Fixes
+
+* Show second for heatmap cell hover timestamp ([#327](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/327))
+* Fix sunburst chart height is 0 ([#332](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/332))
+
+### Documentation
+
+* Update release notes ([#326](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/326))
```

Comment:
This release note https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/326 seems to be for 1.11 release. Should we include it?

Reply:
Yeah I know, accidentally wasn't included in the last tag, and the release notes will pick up all PRs from the last tag. I'm ok for leaving it or removing it

Reply:
I see. I think we can remove it since such PR is not related to current release. Please merge after removing it.